src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -59,10 +59,13 @@
   "minecolonies.config.listofplantables.comment": "The items and item-tags that the Florist can plant.",
 
   "item.minecolonies.scepterlumberjack": "Forest Tool",
-  "item.minecolonies.scepterlumberjack.usedstart": "First position set.",
-  "item.minecolonies.scepterlumberjack.usedend": "Second position set.",
-  "item.minecolonies.scepterlumberjack.restrictiontoobig": "Area %s is too large. Maximum restricted area for the Forester is %s blocks.",
-  "item.minecolonies.scepterlumberjack.restrictionset": "Forester restricted area set. Size is %s blocks. Max size is %s blocks.",
+  "item.minecolonies.scepterlumberjack.usedstart": "First position set. (Click same position to finish)",
+  "item.minecolonies.scepterlumberjack.usedend": "Second position set. (Click same position to finish)",
+  "item.minecolonies.scepterlumberjack.restrictiontoobig": "An area of %d blocks is too large. The maximum restricted area for the Forester is %d blocks.",
+  "item.minecolonies.scepterlumberjack.restrictionset": "Forester restricted area set (x: %d to %d, z: %d to %d, area: %d blocks). The max size is %d blocks.",
+  
+  "minecolonies.config.maxtreesize": "Max tree size",
+  "minecolonies.config.maxtreesize.comment": "Maximal log count in one tree during lumberjack tree search.",
```

```suggestion
  "minecolonies.config.maxtreesize.comment": "Max log count in one tree for the Forester to check during their tree search.",
```
> It's not the lumberjack anymore :D

By Raven in the old PR